_board/bast_pro_mini_m0.md

```diff
@@ -12,9 +12,29 @@ features:
   - Breadboard-Friendly
 ---
 
-Is she the goddess of cats? 
+The Bast Pro Mini M0 features a powerful SAMD21E chip, an ARM Cortex-M0 based microcontroller with low power consumption and high performance, makes it ideal for endless applications.
 
-It’s Bast Pro Mini M0! With a powerful SAMD21E chip, an ARM Cortex-M0 based microcontroller with low power consumption and high performance, makes it ideal for endless applications.
+## Technical details
+
+* Microcontroller Arm Cortex-M0 ATSAMD21E18A with 48 MHz
+* Flash Memory 256 KB
+* SRAM Memory 32 KB
+* Operating Voltage	1.62 V – 3.63 V
+* Input Voltage 5 V (recommended)
+* 12 Digital I/O Pins
+* 10 PWM Digital I/O Pins
+* 6 Analog Input Pins
+* DC Current per I/O Pin 10 mA
```

Comment:
Did you count these up, or are these the chip specifications?

Reply:
That's also coming out of their "Tech Specs" listing.